### PR TITLE
logging improvements

### PR DESCRIPTION
**Pull request by mmaxim**
I dropped some chat logs, and demoted some log lines from feature cache and hidden team loader which showed up on profiles.

cc @maxtaco @AMarcedone

### CODE REVIEW

**Comment by AMarcedone** [comment]
LGTM for the feature cache and hidden team loader lines, they are not very important

File: go/libkb/globals.go [hunk 1]
```diff
@@ -372,7 +372,10 @@ func (g *GlobalContext) ConfigureLogging(usage *Usage) error {
 	// Start redirecting logs if the logFile should be used
 	// If this is not called, prints logs to stdout.
 	if ok {
-		err := logger.SetLogFileConfig(g.Env.GetLogFileConfig(logFile))
+		err := logger.SetLogFileConfig(g.Env.GetLogFileConfig(logFile), &logger.BufferedLoggerConfig{
+			Frequency: time.Second,
+			Size:      1000000,
```
Comment (jakob223) [inline go/libkb/globals.go:377]: A second is a lot, we may just entirely lose what caused a crash. Might be worth having a conversation in core about it. I'm surprised you don't get as-good results with like 200ms.

File: go/chat/localizer.go [hunk 2]
```diff
@@ -1012,15 +984,9 @@ func (s *localizerPipeline) localizeConversation(ctx context.Context, uid gregor
 	case chat1.ConversationMembersType_TEAM:
 		// do nothing
 	case chat1.ConversationMembersType_IMPTEAMNATIVE, chat1.ConversationMembersType_IMPTEAMUPGRADE:
-		public := conversationLocal.Info.Visibility == keybase1.TLFVisibility_PUBLIC
-		resetUsernamesPegboard, err := s.getResetUsernamesPegboard(ctx, umapper, membersType, info, public)
```
Comment (jakob223) [inline go/chat/localizer.go:1016]: I don't really understand what was going on here or why it was unneeded.

Comment (mmaxim) [reply go/chat/localizer.go:1016]: An unused feature

File: go/libkb/globals.go (hunk 1 above)
Comment (mmaxim) [reply go/libkb/globals.go:377]: Sure, I'll make it 200ms, the key change here is the buffer size anyway.

Comment (jakob223) [reply go/libkb/globals.go:377]: If a second was arbitrary, can you test even smaller? we should go as small as we can without impacting performance.

**Pull request merged**